jsApp/src/jsMain/kotlin/main.js.kt

```diff
@@ -20,18 +23,30 @@ fun main() {
         BrowserViewportWindow("Compose Modifiers Playground") {
             var activeTemplate by remember { mutableStateOf(Templates.Rainbow) }
 
+            // Note: isSystemInDarkTheme() is not yet implemented in JS. It will always return false until
+            // the feature is implemented.
+            val isSystemInDarkTheme = isSystemInDarkTheme()
+            var useDarkTheme by remember { mutableStateOf(isSystemInDarkTheme) }
+            val themeColors = if (useDarkTheme) appDarkColors else appLightColors
+
             // Decrease the touch slop. The default value of too high for desktop
             val vc = LocalViewConfiguration.current.withTouchSlop(
                 with(LocalDensity.current) { 0.125.dp.toPx() },
             )
 
             CompositionLocalProvider(LocalViewConfiguration provides vc) {
-                PlaygroundTheme {
+                PlaygroundTheme(colors = themeColors) {
                     Playground(
                         modifier = Modifier.fillMaxSize(),
                         activeTemplate = activeTemplate,
                         onTemplateChange = {
                             activeTemplate = it.copy()
+                        },
+                        darkModeSupported = true,
+                        darkMode = useDarkTheme,
+                        onDarkModeChange = {
+                            // TODO: Find out why toggling dark mode on or off doesn't work
```

Comment:
I think I was able isolate what the issue is. It's with TextField. You can repro with this snippet:

<img width="692" alt="image" src="https://github.com/c5inco/Compose-Modifiers-Playground/assets/1253402/92f9e55b-4a8a-4a80-8b69-3e3dd294ea52">

So likely a thing to go investigate with JetBrains.

Reply:
This is quite surprising. I recreated something similar to your example. From what I was able to test, the issue with TextField only happened after I added a Surface. But then I replaced the Surface with the `.background` modifier on Column, and this specific issue didn't seem to happen anymore. There are other issues, but toggling still works nonetheless when using `.background` for some reason.